### DISCUSSION TITLE
For 42140, how to realize the reverse of the HUB 180 degrees, the same key is pressed, the electric machine will rotate in reverse?

### DISCUSSION THREAD

**Discussion by myhewen** [pybricks/support#1581] [Q&A]
For example, when the light side of the HUB is facing up, press the A key, and the motor is rotating clockwise, but when the light side of the HUB is facing down, press the A key, and the motor is rotating counterclockwise

Can you give me a sample picture of the code block Settings? Thank you!

**Comment by laurensvalk** [comment]
When the hub is upside down, the z acceleration is negative. So you can do something like this.

The hub light is green normally, and red when upside down:

![image](https://github.com/pybricks/support/assets/12326241/d58d9759-17ae-41b1-8ea0-8a14a2e843a1)

**Comment by myhewen** [comment]
thanks！
